build/android/android.gni

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
I don't see why this file is not being added to the same directory as `linux.gni`.